DQM/HcalMonitorTasks/src/HcalDigiMonitor.cc

```diff
@@ -570,6 +570,8 @@ void HcalDigiMonitor::analyze(edm::Event const&e, edm::EventSetup const&s)
 		  i<=FEDNumbering::MAXHCALuTCAFEDID; i++) {
 	  if (i>FEDNumbering::MAXHCALFEDID && i<FEDNumbering::MINHCALuTCAFEDID)
 		  continue;
+	  if (i<1118)
```

Comment:
also, shouldn't this be ``` if (i > MAXHCALFEDID && i < 1118 ) ```
if I understand correctly, now only HF uTCA FEDs will pass.